internal/nginx/manager.go

```diff
@@ -37,6 +37,10 @@ const (
 
 	// appProtectLogConfigFileName is the location of the NGINX App Protect logging configuration file
 	appProtectLogConfigFileName = "/etc/app_protect/bd/logger.cfg"
+
+	appProtectDosAgentInstallCmd    = "/usr/bin/adminstall"
+	appProtectDosAgentStartCmd      = "/usr/bin/admd -d --standalone > /var/log/adm/admd.log 2>&1"
```

Comment:
how big the log file can grow?
if it can grow big, is there a log rotation mechanism?
usually, we expose logs through stdout/stderr of the container

Reply:
Will expose through stdout of the container